packages/nodes-base/nodes/Amqp/AmqpTrigger.node.ts

```diff
@@ -257,11 +263,16 @@ export class AmqpTrigger implements INodeType {
 
 		// The "closeFunction" function gets called by n8n whenever
 		// the workflow gets deactivated and can so clean up.
-		async function closeFunction() {
-			container.removeAllListeners('receiver_open');
-			container.removeAllListeners('message');
-			connection.close();
-		}
+		const closeFunction = async () => {
+			try {
+				container.removeAllListeners('receiver_open');
+				container.removeAllListeners('message');
+				connection.close();
+			} catch (error) {
+				// eslint-disable-next-line n8n-nodes-base/node-execute-block-wrong-error-thrown
+				throw new TriggerCloseError(this.getNode(), { cause: error as Error, level: 'warning' });
```

Comment:
Am I right in thinking `TriggerCloseError` will always be `warning`? If so, let set this in the error?

Reply:
@netroy what do you think? since you introduced TriggerCloseError

Reply:
Let's make all it warning by default. for the vast majority of these errors, we don't need to send these to Sentry.